src/main/java/Task.java

```diff
@@ -0,0 +1,37 @@
+public class Task {
+    protected String description;
+    protected String name;
+    protected boolean isDone = false;
+
+
+    public Task(String description, String name) {
+        this.description = description;
+        this.name = name;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "X" : " "); // mark done task with X
+    }
+
+    public String getDescription() {
+        return this.description;
+    }
+
+    public void markAsDone() {
+        this.isDone = true;
+    }
+
+    public void markAsNotDone() {
+        this.isDone = false;
+    }
+
+    public String name() {
+        return this.name;
+    }
+
+    @Override
+    public String toString() {
+        return "[" + getStatusIcon() + "] " + getDescription();
```

Comment:
Is getDescription() necessary here? 🤔 
```suggestion
        return "[" + getStatusIcon() + "] " + description;
```